frontend/src/models/types.ts

```diff
@@ -31,3 +31,9 @@ export type DateRangeProperties = {
   oldestYearMonth: number[],
   newestYearMonth: number[],
 }
+
+export type Annotations = {
```

Comment:
Small thing: I would make these fields not nullable and just default initialized them as empty arrays.

It will avoid us later needing to make tedious if else checks to verify the array is not null before using them :) .

Reply:
Makes sense! changes are pushed